WordPress/src/androidTest/java/org/wordpress/android/e2e/flows/LoginFlow.kt

```diff
@@ -67,10 +67,7 @@ class LoginFlow {
         if (!isSelfHosted) {
             // Quick Start Prompt Dialog - Click the "No thanks" negative button to continue.
             // See QuickStartPromptDialogFragment
-            val negativeButton =
-                Espresso.onView(ViewMatchers.withId(R.id.quick_start_prompt_dialog_button_negative))
-            WPSupportUtils.waitForElementToBeDisplayed(negativeButton)
-            WPSupportUtils.clickOn(negativeButton)
```

Comment:
`clickOn` already contains `waitForElementToBeDisplayed`, so it was redundant here. Also, I went for passing the identifer to the function, instead of the ViewMatcher, to keep it shorter.